libhoth_usb.c

```diff
@@ -98,10 +98,6 @@ int libhoth_usb_open(const struct libhoth_usb_device_init_options* options,
   if (status != LIBUSB_SUCCESS) {
     goto err_out;
   }
-  status = libusb_claim_interface(dev->handle, info.interface_id);
```

Comment:
This will break what the hothd folks are currently doing. Here's how I was originally thinking this should work so that it doesn't change the API semantics:

- libusb_claim_interface() increments a reference count. If the previous reference count was zero, it asks the kernel to claim the interface

- libusb_release_interface() decrements a reference count, and when zero asks the kernel to release the interface.

- libhoth_usb_send_request() calls libusb_claim_interface() (plus libusb_release_interface() on error), and libhoth_usb_receive_response() calls libusb_release_interface after it has received the response.